core/templates/footer.html

```diff
@@ -11,7 +11,9 @@
                     </span>
                     <span class="tel">+1-650-318-1652</span>
                 </p>
-                <p>Western Friend is published by the Friends Bulletin Corporation, an independent 501(c)3 non-profit (EIN-93-1156843), established jointly by Pacific, North Pacific, and Intermountain Yearly Meetings</p>
+                <p>Western Friend is published by the Friends Bulletin Corporation, an independent 501(c)3 non-profit (EIN-93-1156843), 
+                    established jointly by Pacific, North Pacific, and Intermountain Yearly Meetings. This website is <a href="https://github.com/WesternFriend/WF-website" target="_blank" class="link-secondary">Open Source on GitHub</a>.
+                </p>
```

Comment:
These can be separate paragraphs.
```suggestion
                <p>Western Friend is published by the Friends Bulletin Corporation, an independent 501(c)3 non-profit (EIN-93-1156843),  established jointly by Pacific, North Pacific, and Intermountain Yearly Meetings.</p>
                    
                    <p>This website is <a href="https://github.com/WesternFriend/WF-website" target="_blank" class="link-secondary">Open Source on GitHub</a>.
                </p>
```